crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -419,26 +419,38 @@ type Epoch {
 
 type Event {
 	"""
-	Package, module, and type of the event
+	The Move module that the event was emitted in.
```

Comment:
Can you clarify whether this was the module that contains the function that called `event::emit` or the module containing the entrypoint function (the one that was called by the PTB's MoveCall).  Another way of asking the question: Is this the top of the stacktrace or the bottom of the stacktrace at the point `event::emit` is called.

Reply:
Wrote a test case to validate this. The emitting module is the module containing the entrypoint function (top of the stacktrace)

Reply:
Just to be absolutely sure, the entrypoint function being the one that is closest to the PTB, and not the one that is closest to the `event::emit` right? (For me, stacks grow up so the entrypoint is the bottom of the stack 😅).

Reply:
yes the fn closest/ called by the PTB. and ah right first in last out :painlaugh:

Reply:
It's a legitimate take to say that stacks grow down -- stacks in assembly grow down -- so the main comment here is to not use up/down top/bottom to distinguish the side of a stack in the docs for this field, and I regret using those terms in my initial PR comment!